### PR TITLE
BoilerPlate setup with truffle Drizzle

### PR DESCRIPTION
**Pull request by avp1598**
BoilerPlate setup with truffle Drizzle using truffle unbox drizzle.

### CODE REVIEW

File: contracts/TutorialToken.sol [hunk 1]
```diff
@@ -0,0 +1,15 @@
+// SPDX-License-Identifier: MIT
+pragma solidity >=0.4.21 <0.7.0;
+
+import "@openzeppelin/contracts/token/ERC20/ERC20.sol";
+
+contract TutorialToken is ERC20 {
```
[the remark this answers is not in the document]
Comment (avp1598) [reply contracts/TutorialToken.sol:6]: Just a placeholder for our ERC20 Token

File: contracts/ComplexStorage.sol [hunk 2]
```diff
@@ -0,0 +1,44 @@
+// SPDX-License-Identifier: MIT
+pragma solidity >=0.4.21 <0.7.0;
+
+contract ComplexStorage {
```
[the remark this answers is not in the document]
Comment (avp1598) [reply contracts/ComplexStorage.sol:4]: Yeah I'll delete

File: truffle-config.js [hunk 3]
```diff
@@ -0,0 +1,27 @@
+const path = require("path");
+
+module.exports = {
+  // See <http://truffleframework.com/docs/advanced/configuration>
+  // to customize your Truffle configuration!
+  contracts_build_directory: path.join(__dirname, "app/src/contracts"),
+  networks: {
```
[the remark this answers is not in the document]
Comment (avp1598) [reply truffle-config.js:7]: added

File: app/.gitignore [hunk 4]
```diff
@@ -0,0 +1,23 @@
+# See https://help.github.com/articles/ignoring-files/ for more about ignoring files.
```
[the remark this answers is not in the document]
Comment (avp1598) [reply app/.gitignore:1]: done

**Comment by adityachakra16** [comment]
Merge merge

**Pull request merged**